subsys/emul/spi/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
I think it makes more sense to do:

    zephyr_include_directories_ifdef(CONFIG_EMUL_BMI160 ${ZEPHYR_BASE}/drivers/sensor)

than to move the private driver header into the global include directory.

Reply:
Does that mean:

zephyr_include_directories_ifdef(CONFIG_EMUL_BMI160 ${ZEPHYR_BASE}/drivers/sensor/bmi160)

and then using:

#include <bmi160.h>

in the emulator? If so, it works, but it seems confusing to include bmi160.h as a header file that I would expect to find in include/ but which is actually in drivers/sensor/bmi160/

Reply:
Yes, that's what I meant.  I don't think you should expect to find headers that are intended for driver implementation sources (or emulator implementation sources) in the system include area.  The main include area should be for things that applications are expected to include, and not be cluttered with things they are not expected to include.

Reply:
OK done